chain/epoch_manager/src/lib.rs

```diff
@@ -135,18 +140,20 @@ impl EpochManager {
             if !validator_kickout.contains(&account_id) {
                 validator_online_ratio.insert(
                     account_id.clone(),
-                    (
-                        num_blocks * total_chunks_expected
-                            + expected_blocks * total_chunks_produced,
-                        total_chunks_expected * expected_blocks * 2,
-                    ),
+                    ValidatorStats {
```

Comment:
why do you use `ValidatorStats` here? It might be confusing because it is expected that fields in `ValidatorStats` reflects what they are supposed to mean.

Reply:
after re-reading this, I'm not sure I understand fully why validator_online_ratio is computed this way:
```
                    (
                        num_blocks * total_chunks_expected
                            + expected_blocks * total_chunks_produced,
                        total_chunks_expected * expected_blocks * 2,
                    ),
```

Is it suppose to be `num_blocks / expected_blocks + total_chunks_produced / total_chunks_expected`

where is 2 coming from?